services/ui-src/src/components/reports/ReportProvider.tsx

```diff
@@ -108,6 +110,16 @@ export const ReportProvider = ({ children }: Props) => {
     }
   };
 
+  const unlockReport = async (reportKeys: ReportKeys) => {
+    try {
+      const result = await unlockReportRequest(reportKeys);
+      setReport(result);
+      setLastSavedTime(getLocalHourMinuteTime());
```

Comment:
Comment: I don't think we need to set the autosave time when a report is being released back to the state users.